src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User Bohdan = new User("Bohdan", "password", 58);
+    private static final User Dan = new User("Dan", "NothingPassword", 68);
+    private User defaultUser;
+    private RegistrationService service;
+
+    @BeforeEach
+    void getService() {
+        RegistrationService service = new RegistrationServiceImpl();
+        User defaultUser = new User("artemk", "12345678", 18);
+    }
+
+    @Test
+    void setNullUser_notOk() {
+        assertThrows(RuntimeException.class, () -> {
+            service.register(null);
+        });
+        assertTrue(Storage.people.isEmpty());
```

Comment:
you don’t need to check Storage in this case it does not demonstrate that the method register works as expected. But if you want check it you need clear storage after even each test case